R/helpers.R

```diff
@@ -621,3 +621,47 @@ read_zip_shapefile <- function(mn, pid){
   unlink(temp)
   return(shapefile)
 }
+
+
+#' Recovers failed submissions
+#'
+#' Recovers failed submissions and write the new, valid EML to a given path
+#'
+#' @param node (MNode) The Member Node to publish the object to.
+#' @param pid The PID of the EML metadata document to be recovered.
+#' @param path path to write XML
+#'
+#' @return recovers and write the valid EML to the indicated path
+#'
+#' @export
+#'
+#' @author Rachel Sun rachelsun@@ucsb.edu
+#'
+#' @examples
+#' \dontrun{
+#' # Set environment
+#' cn <- CNode("STAGING2")
+#' mn <- getMNode(cn,"urn:node:mnTestKNB")
+#' pid <- "urn:uuid:c40b93d9-a15a-47d0-9a5f-06e7056c93c1"
+#' path <- system.file("extdata", "example.xml", package = "emld")
```

Comment:
This is a good thought but I think you're using `system.file` a bit incorrectly here.  That function searches for files in R packages.  Since we're writing a file to a path you'd want to use either `tempdir()` and a file name, or just `tempfile()`.